packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -541,14 +541,21 @@
                             >
                             </span>
 
-                            <p class="m-0" data-ng-if="$ctrl.vpsLinkedDisk.size">
-                                <span data-translate="vps_additional_disk_current_capacity"
-                                      data-translate-values="{vpsDiskCapacity: ($ctrl.vpsLinkedDisk.size | bytes:0:false:'GB')}">
+                            <p
+                                class="m-0"
+                                data-ng-if="$ctrl.vpsLinkedDisk.size"
+                            >
+                                <span
+                                    data-translate="vps_additional_disk_current_capacity"
+                                    data-translate-values="{vpsDiskCapacity: ($ctrl.vpsLinkedDisk.size | bytes:0:false:'GB')}"
+                                >
                                 </span>
-                                <span class="oui-icon oui-icon-help-circle"
-                                      aria-hidden="true"
-                                      data-ng-if="!$ctrl.isVpsNewRange"
-                                      oui-tooltip="{{:: 'vps_additional_disk_cant_upgrade' | translate}}">
+                                <span
+                                    class="oui-icon oui-icon-help-circle"
+                                    aria-hidden="true"
```

Comment:
Maybe we need a sr-only span because tooltip won't be accessible due to aria-hidden="true"

Reply:
This change is only appearing only because of the formatting. So I feel maybe we can handle this with a different ticket. What do you suggest @marie-j ?

Reply:
I was mislead by the UI, fine if we treat it separately